GameData/RP-1/Contracts/Commercial Applications 1/FirstComSat-CA.cfg

```diff
@@ -0,0 +1,131 @@
+CONTRACT_TYPE
+{
+	name = FirstComSat-CA
+	title = First Communications Satellite
+	group = CommApp
+	agent = Federation Aeronautique Internationale
+
+	description = <b>Program: Early Commercial Applications<br>Type: <color=green>Required</color></b><br><br>Our communications around the globe are limited to line of sight because of the curvature of Earth. If we can launch a communications satellite into orbit, we can use it to transmit information to the opposite side of the world.<br><br>Historical example: Telstar 1 (78kg, Thor-Delta)
+
+	synopsis = Launch the first communication satellite
+
+	completedMessage = Congratulations! Satellite communications will forever change communications around the globe.
+
+	sortKey = 512
+
+	cancellable = true
+	declinable = false
+	autoAccept = false
+	minExpiry = 0
+	maxExpiry = 0
+	maxCompletions = 1
+	maxSimultaneous = 1
+	deadline = 0
+
+	targetBody = HomeWorld()
+
+
+	// ************ REWARDS ************
+	prestige = Trivial       // 1.0x
+	advanceFunds = 0
+	rewardScience = 0
+	rewardFunds = 0
+	failureFunds = 0
+	rewardReputation = 45
+	failureReputation = 0 // was @rewardReputation
+
+	// ************ REQUIREMENTS ************
+
+	REQUIREMENT
+	{
+		name = ProgramActive
+		type = ProgramActive
+		program = EarlyCommApp
+	}
+	
+	DATA
+	{
+		type = float
+		payload = 80
+	}
+
+	PARAMETER
+	{
+		name = FirstComSat
+		type = VesselParameterGroup
+		title = First Communications Satellite
+		define = FirstComSat
+		
+		PARAMETER
+		{
+			name = NewVessel
+			type = NewVessel
+			title = Launch a new vessel
+			hideChildren = true
+		}
+		PARAMETER 
+		{
+			name = Crewmembers
+			type = HasCrew
+			minCrew = 0
+			maxCrew = 0
+			title = Uncrewed
+			hideChildren = true
+		}
+		PARAMETER
+		{
+			name = HasComSatPayload
+			type = HasResource
+			resource = ComSatPayload
+			minQuantity = @/payload-0.1
+			title = Have a ComSatPayload of at least @/payload units on the craft
+			hideChildren = true
+		}
+		PARAMETER
+		{
+			name = Orbit
+			type = Orbit
+			minInclination = 40
+			minPeA = 850000
+			minApA = 5000000
+			disableOnStateChange = true
+			title = Achieve orbit with a minimum inclination of 40 degrees, a minimum perigee of 850 km, and an apogee of more than 5,000 km
+			
+			PARAMETER
+			{
+				name = Duration
+				type = Duration
+
+				duration = 30d
```

Comment:
This still has the 30d duration requirement. All the contracts in CA should have duration requirements of 2 minutes, like they had in Targeted Sats, except the network ones that should keep their 2 day requirements from the template network contracts.

Reply:
Caught that as I was scrolling through the changed files.